recipe/conda_build_config.yaml

```diff
@@ -46,6 +46,10 @@ zip_keys:
 pin_run_as_build:
   arpack:
     max_pin: x.x
+  boost:
+    max_pin: x.x.x
```

Comment:
How about just `boost-cpp`?.

Reply:
What about Python/C++ binding code that uses Boost?

Reply:
This is why we have split up the boost package.  We have libboost for explicitly compiling things against boost.  We have py-boost for python usage.  Ideally, I think we'd also have a boost-includes or something like that for strictly header-only components.

With current conda-forge recipes, I believe it is safest to use the setting you have here, but I would encourage everyone to move towards packages that are only used one way (linking, or python, or only headers).

Reply:
I though boost python was a C++ library and not a python package.

Reply:
You are right.  We split it up so that we only build the core libraries once, then compile only the python part by itself.  So yes, you are correct to have pin_run_as_build for boost and boost-cpp, and we would need it for libboost and py-boost.